src/components/inputs/Label/Label.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import webexComponentClasses from '../../helpers';
+import Icon from '../../generic/Icon/Icon';
+
+/**
+ * InputController
```

Comment:
Label

and a short description eg: wraps a form input and displays label text and error message

and an empty line before params